### DISCUSSION TITLE
The effect of converting pictures in other formats to svg format is not good

### DISCUSSION THREAD

**Discussion by charliecen** [ImageMagick/ImageMagick#6006] [Help] [answered]
### ImageMagick version

7.1.0-58 

### Operating system, version and so on

Alpine Linux v3.15

### Description

https://github.com/ImageMagick/ImageMagick/issues/5968#issuecomment-1397853948

**Comment by dlemstra** [comment, marked as answer]
Locking this conversation. We have seen your comment but haven't had time to react to it yet. There is no need to start a new conversion.
